### PR TITLE
Implement Forgot Password Screen #6

### PR DESCRIPTION
**Pull request by abdurahman-ctis**
Implement Forgot Password Screen #6
- Implemented Forgot Password Screen
Capture
## Type of Changes?
Just put an x in the [ ] which are valid.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing Suites

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes.
- [ ] ./gradlew assembleDebug assembleRelease

## Checklist:
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by iadeelzafar** [comment]
@abdurahman-ctis Thanks for working on this ticket. please add a gif/screenshot on the PR description showing changes.

File: app/src/main/res/drawable/edittext_rounded.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+android:shape="rectangle">
+
+<stroke
+    android:width="2dp"
+    android:color="#1b9dd9" />
```
Comment (AminullahTajMuhammad) [inline app/src/main/res/drawable/edittext_rounded.xml]: add this code in colors.xml

File: app/src/main/res/layout/activity_forgot_password.xml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        android:id="@+id/titleText"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="91dp"
+        android:layout_marginLeft="91dp"
+        android:layout_marginTop="144dp"
+        android:layout_marginEnd="91dp"
+        android:layout_marginRight="91dp"
+        android:text="@string/forget_password"
+        android:textColor="#1b9dd9"
```
Comment (AminullahTajMuhammad) [inline app/src/main/res/layout/activity_forgot_password.xml]: add this in color code in color.xml

File: app/src/main/res/layout/activity_forgot_password.xml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        android:layout_marginStart="34dp"
+        android:layout_marginLeft="34dp"
+        android:layout_marginTop="21dp"
+        android:layout_marginEnd="28dp"
+        android:layout_marginRight="28dp"
+        android:layout_marginBottom="228dp"
+        android:background="@drawable/button_custom"
+        android:text="@string/send"
+        android:textAllCaps="false"
+        android:textColor="#FFFFFF"
```
Comment (AminullahTajMuhammad) [inline app/src/main/res/layout/activity_forgot_password.xml]: add color code in color.xml

**Comment by abdurahman-ctis** [comment]
Did I understand correctly, I should take all colors(hex codes) and name them in the `colors.xml`

**Comment by AminullahTajMuhammad** [comment]
@abdurahman-ctis yep

**Comment by AminullahTajMuhammad** [comment]
@abdurahman-ctis Any update?

**Comment by iadeelzafar** [comment]
@abdurahman-ctis  please make the asked changes on this ticket so that we can merge this PR.

**Comment by pfieffer** [comment]
Can I work on this or is it already nearly completed?

**Comment by abdurahman-ctis** [comment]
@pfieffer you can, I actually don't have time for it

**Comment by iadeelzafar** [comment]
Implemented in #21

**Pull request closed without merging**